client/src/components/QueryComponent/QueryComponent.js

```diff
@@ -35,13 +35,27 @@ class QueryComponent extends Component {
       showConfirmDeleteModal: false,
       showConfirmOverwriteModal: false,
       newQueryName: "",
-      isOverlayResultsChecked: false
+      isOverlayResultsChecked: false,
+      multiline: false,
+      multilineHolder: false
     };
   }
 
   componentDidMount() {
     this.setState({ queries: settingsService.queries });
     eventService.subscribeEnvironmentChange(this.clearAfterEnvironmentChange);
+    document.addEventListener("keyup", this.upFunction, false);
+    document.addEventListener("click", this.onFocusLost, false);
+  }
+
+  upFunction = event => {
+    const enterPressed = event.keyCode === 13;
+    if (event.shiftKey) {
+      if (enterPressed) {
```

Comment:
We can probably collapse these two if statements into a single clause. I.e. `event.shiftKey && enterPressed`

Reply:
Changed to Key instead